app/src/js/templates.js

```diff
@@ -11,56 +11,62 @@ export const reportHeadingTemplate = function(data) {
-                <svg class="usa-icon" aria-labelledby="fork-count-${data.name}-title" role="img">
-                <title id="github-${data.name}-title"> Fork Count</title>
-                <svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-git-fork"><circle cx="12" cy="18" r="3"/><circle cx="6" cy="6" r="3"/><circle cx="18" cy="6" r="3"/><path d="M18 9v2c0 .6-.4 1-1 1H7c-.6 0-1-.4-1-1V9"/><path d="M12 12v3"/></svg>
-                </svg>
-                <span> ${ data.forks_count } </span>
-            </span>
-            <span class="issue-count" style="margin-right: 15px;">
-                <svg class="usa-icon" aria-labelledby="issue-count-${ data.name }-title" role="img">
-                <title id="github-{{data.name}}-title"> Issue Count</title>
-                <svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-circle-dot"><circle cx="12" cy="12" r="10"/><circle cx="12" cy="12" r="1"/></svg>
-                </svg>
-                <span> ${ data.issues_count } </span>
-            </span>
-            <span class="pull-request-count" style="margin-right: 15px;">
-                <svg class="usa-icon" aria-labelledby="pull-request-count-${data.name}-title" role="img">
-                <title id="github-${data.name}-title"> Pull Request Count</title>
-                <svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-git-pull-request"><circle cx="18" cy="18" r="3"/><circle cx="6" cy="6" r="3"/><path d="M13 6h3a2 2 0 0 1 2 2v7"/><line x1="6" x2="6" y1="9" y2="21"/></svg>
-                </svg>
-                <span> ${ data.pull_requests_count } </span>
-            </span>
-            <span class="watchers-count">
-                <svg class="usa-icon" aria-labelledby="watchers-count-${data.name}-title" role="img">
-                <title id="github-${data.name}-title"> Watchers Count</title>
-                <svg xmlns="http://www.w3.org/2000/svg" width="17" height="18" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-eye"><path d="M2.062 12.348a1 1 0 0 1 0-.696 10.75 10.75 0 0 1 19.876 0 1 1 0 0 1 0 .696 10.75 10.75 0 0 1-19.876 0"/><circle cx="12" cy="12" r="3"/></svg>                </svg>
-                <span> ${ data.watchers_count } </span>
-            </span>
+        <div class="usa-card-outer outer-project-template">
+            <div class="usa-card-middle">
+                <a href="${ data.url }/${ data.owner }/${ data.name }/" class="href-home">
```

Comment:
Not sure if the fix is here but the project card link takes the user to the repo on github.com, instead of the project report

Reply:
I believe `data.url` contains the repo's github link so this can be removed!

Reply:
I was able to fix the links to work in production testing, but not in local resting. When I run `npm run prod` they work, when I run `npm start` they do not.

Reply:
I was able to fix the link when it is in prod.  When I run `npm run prod` they work. When I run `npm start` they do not?

Reply:
Gotcha, check out this PR. Here, I added the `baseurl` variable to `data` in the project card template so I think you can use `data.baseurl` https://github.com/DSACMS/metrics/pull/270/files

Reply:
I'm struggling with the conditional of this, for testing.

Reply:
@DinneK I got it!

Line 16 can be changed to `const baseUrl = window.location.hostname.includes("dsacms.github.io") ? '/metrics' : "";`

Reply:
If the hostname is `dsacms.github.io` then we add `/metrics/. If not (aka the hostname is localhost), then we leave it blank

Reply:
Oops i take that back, hmmmm

Reply:
Done! All good now